### ISSUE TITLE
Export board state as FEN

### ISSUE THREAD

**Issue by AndreVallestero** [brettbates/pgn_parser#10]
Please add a feature to export the board state as FEN. This would simplify usage of the project with systems that render the board in using FEN.

**Comment by brettbates** [comment]
Good idea, perhaps Game.as_fen(ply), calculating the FEN up to and including that ply.

https://en.wikipedia.org/wiki/Forsyth%E2%80%93Edwards_Notation

Bit of work, have to calculate the game position which means some coding of how chess plays.
